src/app/(admin)/new_dashboard/guides/page.tsx

```diff
@@ -49,7 +47,9 @@ const Page = () => {
         </Box>
       </DashboardCard>
     </PageContainer>
+    {selectedContact && <ChatPopup selectedContact={selectedContact} />}
+    </>
```

Comment:
Why would we need to insert the component in each page? 

https://en.wikipedia.org/wiki/Don%27t_repeat_yourself